README.md

```diff
@@ -47,7 +47,7 @@ pr: https://github.com/elastic/elastic-agent/pull/823
 Prerequisites:
 - installed [mage](https://github.com/magefile/mage)
 - [Docker](https://docs.docker.com/get-docker/)
-- [X-pack](https://github.com/elastic/beats/tree/main/x-pack) to pre-exist in the parent folder of the local Git repository checkout
+- [beats](https://github.com/elastic/beats) to pre-exist at ../beats
```

Comment:
Any reason to change the wording here and not keeping ` in the parent folder of the local Git repository checkout`?

Reply:
it comes from a slack thread. The agent does not need `x-pack` it actually needs `beats` and needs it in the same level as the agent repo itself. I believe it might have been copied from beats and not updated.

This is what we need:

```
your-root-folder
├── agent
└── beats
    └── x-pack
```

Reply:
Yes my point was not about changing X-Pack by Beats but keeping the fact that beats repository should be in the parent folder of the Agent git repository.

Reply:
no worries @AndersonQ I put back the original wording. My concision hammer went too far ;)